packages/patterns-react/src/patterns/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -14,5 +14,40 @@ storiesOf('LeadSpace', module)
     },
   })
   .add('Default', () => {
-    return <LeadSpace />;
+    const copy = text(
+      'copy',
+      'Use this area for a short line of copy to support the title'
+    );
+
+    const title = text('title', 'Lead space title');
+
+    const variations = {
+      expressive: '',
+      productive: 'productive',
+    };
+
+    const imageUrl = text('imageUrl', 'https://picsum.photos/id/1025/1000/400');
+
+    const buttons = [
+      {
+        link: '',
+        copy: 'Primary action',
+        renderArrow: true,
+      },
+      {
+        link: '',
+        copy: 'Secondary action',
+        renderArrow: false,
+      },
+    ];
+
+    return (
+      <LeadSpace
+        title={title}
+        copy={copy}
+        buttons={object('buttons', buttons)}
```

Comment:
I wonder if the designers are going to have difficulty trying to test this out. Any ideas on an easier way to knob this out?

Reply:
I'd say let's hold on this for now, I showed @larahanlon2 and Wonil how to swap out images in storybook so I think they know. In the long term, we are working with Jill and others on what the functional specs for these might be and want to see where they land as far as how these props should be shaped. I still don't know if an array of objects is something that they would want these to be at the end of the day, for example.